doc/api/errors.md

```diff
@@ -221,8 +221,7 @@ Error.captureStackTrace(myObject);
 myObject.stack;  // similar to `new Error().stack`
 ```
 
-The first line of the trace, instead of being prefixed with `ErrorType:
-message`, will be the result of calling `targetObject.toString()`.
+The first line of the trace will be prefixed with `ErrorType.name: message`.
```

Comment:
It will be `myObject.name` if `myObject.message` doesn't exist, or `${myObject.name}: ${myObject.message}` if it does. I think it's the `ErrorType` that's tripping me up a bit.